Documentation/preparing-devel-environment.md

```diff
@@ -0,0 +1,90 @@
+# Preparing development environment and first rkt build
+This is example configuration for fresh instalation of Ubuntu 16.04 GNOME.
+
+### Required software
+You have to install few software packages to correctly build:
+```sh
+$ sudo apt-get install libssl-dev dh-autoreconf libacl1-dev libtspi-dev build-essential
+```
+See also dependencies [page](https://github.com/coreos/rkt/blob/master/Documentation/dependencies.md).
+
+### We definitely recommend using ccache for kvm flavor developers
+Ccache can save a lot of your time. Once build kernel can be taken from cache, so you don't have to build it over and over. You can get and configure ccache in few easy steps:
+
+```sh
+$ sudo apt-get install ccache
+$ ccache --max-size=10G
+$ sudo ln -s /usr/bin/ccache /usr/local/bin/gcc
+$ sudo ln -s /usr/bin/ccache /usr/local/bin/g++
+```
+We set maximum cache size to 10GB. (Default values are too small to perform kernel compilation caching)
+
+### Installing Go Programming Language for single-user
+```
```

Comment:
As this was discussed yesterday - we are using a bit different setup, for single user, without requiring `root` right (access to `/usr/local`) for golang installation.
Still running `rkt` will require `sudo` or plaing `root` account.